app/views/RoomsListView/index.js

```diff
@@ -726,8 +726,7 @@ class RoomsListView extends React.Component {
 	goRoom = ({ item, isMasterDetail }) => {
 		logEvent(events.RL_GO_ROOM);
 		const { item: currentItem } = this.state;
-		const { rooms } = this.props;
-		if (currentItem?.rid === item.rid || rooms?.includes(item.rid)) {
+		if (currentItem?.rid === item.rid) {
```

Comment:
We're removing `rooms` reducer from here, but is it being used anywhere?
Is it needed still?

@reinaldonetof

Reply:
the `rooms` from props are being used at shouldComponentUpdate and componentDidUpdate